waspc/data/Generator/templates/react-app/src/auth/email/actions/login.ts

```diff
@@ -6,7 +6,7 @@ export async function login(data: { email: string; password: string }): Promise<
     try {
         const response = await api.post('{= loginPath =}', data);
         await initSession(response.data.token);
-    } catch (e: unknown) {
+    } catch (e) {
```

Comment:
Type mismatch between `unknown` and `handleApiError`'s argument.

If `handleApiError`'s supposed to be able to catch and handle any error, we should update its signature and implementation.